CONTRIBUTING.md

```diff
@@ -50,12 +50,12 @@ An *actor* is any thing in the game that moves or performs actions or interactio
 
 You should [join the Discord](https://discord.zelda64.dev/) to say hello and get suggestions on where to start on the `#mm-decomp` channel.
 
-We track who is working on what on the [Trello board](https://trello.com/b/ruxw9n6m/majoras-mask-decompilation).
-After joining the Discord, ask to be added to the Trello board so you can claim your code file and avoid duplicate work. You may need to add a card if a card for your file does not currently exist.
+We track who is working on what on the [Google Sheets reservation board](https://docs.google.com/spreadsheets/d/1X83YCPRa532v-Zo0WgUsJ2kB1X9RxBta5_p9aWA8uro).
+After joining the Discord, ask to be added to the Google Sheets reservation board so you can claim your code file and avoid duplicate work. You may need to add a card if a card for your file does not currently exist.
```

Comment:
This looks to be Trello specific instructions. This needs to be updated for the sheets. 

- No longer need to be added by someone
- they aren't cards anymore
- nothing should/can be added except by admins
- maybe perhaps explaining the layout of the sheet briefly
- Other things I'm possibly forgetting.

Reply:
I think I removed Trello specific instructions. I think an explanation of the layout of the sheet is out of scope for this PR, though. It's a good idea, though.